### PR TITLE
Fix load statistics description

### PR DESCRIPTION
**Pull request by stephenc**
Follow up to #1480 

@reviewbybees

### CODE REVIEW

File: core/src/main/resources/hudson/model/LoadStatistics/main.properties [hunk 1]
```diff
@@ -48,5 +56,8 @@ blurb=\
       adding more computers.\
     </dd>\
   </dl>\
-  The graph is exponential moving average of periodically collected data values. \
-  3 timespans are updated every 10 seconds, 1 minute, and 1 hour respectively.
+  <p><b>Note:</b>The number of busy executors and the number of available executors need not \
+  necessarily be equal to the number of online executors as executors can be suspended from\
+  accepting builds and thus be neither busy nor available.</p>\
+  <p>The graph is exponential moving average of periodically collected data values. \
```
Comment (jtnord) [inline core/src/main/resources/hudson/model/LoadStatistics/main.properties]: is *an* exponential moving average?  (not introduced in this commit)

File: core/src/main/resources/hudson/model/LoadStatistics/main.properties [hunk 2]
```diff
@@ -48,5 +56,8 @@ blurb=\
...
+  3 timespans are updated every 10 seconds, 1 minute, and 1 hour respectively.</p>
```
Comment (jtnord) [inline core/src/main/resources/hudson/model/LoadStatistics/main.properties]: no comma before and (not introduced in this commit)

**Comment by jtnord** [comment]
:+1: as my comments are on code not introduced in this change - however would be nice if they got fixed whilst you are working in this area.

File: core/src/main/resources/hudson/model/LoadStatistics/main.properties [hunk 3]
```diff
@@ -22,22 +22,30 @@
...
   <dl> \
-    <dt>Total number of executors</dt> \
+    <dt>Number of online executors</dt> \
     <dd>\
-      For a computer, this is the number of executors that the computer has. \
-      For a label, this is the sum of all executors across all computers in this label. \
-      For the entire Jenkins, this is the sum of all executors across all computers in this Jenkins installation. \
-      Other than configuration changes, this value can also change when slaves go offline. \
+      For a computer, if the computer is online then this is the number of executors that \
+      the computer has, if the computer is offline then this is zero. \
```
Comment (tfennelly) [inline core/src/main/resources/hudson/model/LoadStatistics/main.properties]: > ", if the computer is offline"

Shouldn't that be a new sentence?

**Comment by tfennelly** [comment]
:+1:

File: core/src/main/resources/hudson/model/LoadStatistics/main.properties (hunk 3 above)
Comment (daniel-beck) [inline core/src/main/resources/hudson/model/LoadStatistics/main.properties]: https://en.wikipedia.org/wiki/Comma_splice ?

**Pull request merged**